appeals/web/src/server/appeals/appeal-details/representations/lpa-statement/redact/redact.controller.js

```diff
@@ -0,0 +1,56 @@
+import { patchRepresentationRedaction } from '#lib/api/representation.api.js';
+import { addNotificationBannerToSession } from '#lib/session-utilities.js';
+import { render } from '../../common/render.js';
+import { redactLpaStatementPage, redactConfirmPage } from './redact.mapper.js';
+import { setRepresentationStatus } from '../../representations.service.js';
+
+export const renderRedact = render(redactLpaStatementPage, 'patterns/display-page.pattern.njk');
```

Comment:
I'm still not a fan of these abstractions, although I recognise you're just building on what's already in place and this PR isn't the place to discuss it, I thought it was worth noting